hooks/pre/10-reset_feature.rb

```diff
@@ -25,7 +33,11 @@ def stop_services
 
 def reset_database
   Kafo::KafoConfigure.logger.info 'Dropping database!'
-  Kafo::Helpers.execute('foreman-rake db:drop 2>&1')
+  if @db_config && @db_config.remote_db?(@db_config.foreman)
+    @db_config.empty_database!(@db_config.foreman)
```

Comment:
This is me thinking out loud. There is ```db:purge``` since Rails 4.2 so I wonder if we should try to use that instead. On the other hand, we already need it because candlepin has no equivalent. Let's keep this version.

Reply:
@ekohl thanks for pointing the `db:purge` out. It didn't came to my mind. I'm okay to use it for the foreman db if that helps. No strong opinion though.